DFC.App.DiscoverSkillsCareers/Properties/launchSettings.json

```diff
@@ -11,7 +11,7 @@
     "IIS Express": {
       "commandName": "IISExpress",
       "launchBrowser": true,
-      "launchUrl": "https://localhost:44342/discover-your-skills-careers",
+      "launchUrl": "https://localhost:44342/discover-your-skills-and-careers",

```

Comment:
Can you get "discover-your-skills-and-careers" from @RouteName.Prefix

Reply:
this is a json file.